api/configuration/references.ts

```diff
@@ -60,6 +60,10 @@ export const JURISDICTIONS = 'jurisdictions'
 
 export const IDAM_SECRET = 'secrets.rpx.mc-idam-client-secret'
 
+export const SYSTEM_USER_NAME = 'secrets.rpx.system-user-name'
```

Comment:
Same comment as above about this being a generic name...

Reply:
we want it to be generic. This is a system user name which we are going to use it for several roles/services.